microceph/cmd/microceph/enable_rgw.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+
+	"github.com/canonical/microcluster/microcluster"
+	"github.com/spf13/cobra"
+
+	"github.com/canonical/microceph/microceph/api/types"
+	"github.com/canonical/microceph/microceph/client"
+)
+
+type cmdEnableRGW struct {
+	common   *CmdControl
+	flagPort int
+}
+
+func (c *cmdEnableRGW) Command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rgw [--port <port>]",
+		Short: "Enable the RGW service on this node",
```

Comment:
`node` is a term which hasn't been used in this codebase yet, maybe you should use `server` instead?

Can you also make it so `enable` and `disable` can be used through `UseTarget` with a `--target` flag?

So running `microceph enable rgw --target foo` will call it with `UseTarget("foo")` which will then cause microceph to forward the request to the correct server. This enables much easier management of the cluster by not having to SSH into the right system.